src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaControllerTest.java

```diff
@@ -162,9 +169,39 @@ public void readRcaEnabledFromConf() throws IOException {
     Assert.assertTrue(RcaController.isRcaEnabled());
   }
 
+  @Test
+  public void readAndUpdateMutesRcasBeforeGraphCreation() throws Exception {
+    Method readAndUpdateMutesRcas = rcaController.getClass()
+            .getDeclaredMethod("readAndUpdateMutesRcas", null);
+    readAndUpdateMutesRcas.setAccessible(true);
+
+    String rcaConfPath = Paths.get(RcaConsts.TEST_CONFIG_PATH, "rca_muted.conf").toString();
+    Field rcaConfField = rcaController.getClass().getDeclaredField("rcaConf");
+    rcaConfField.setAccessible(true);
+    rcaConfField.set(rcaController, new RcaConf(rcaConfPath));
+    updateConfFileForMutedRcas(rcaConfPath, Arrays.asList("CPU_Utilization", "Heap_AllocRate"));
+
+    Field mutedGraphNodesField = Stats.class.getDeclaredField("mutedGraphNodes");
+    mutedGraphNodesField.setAccessible(true);
+    mutedGraphNodesField.set(Stats.getInstance(), null);
+    Set<String> initialComponentSet = ConnectedComponent.getNodeNames();
+    Whitebox.setInternalState(ConnectedComponent.class, "nodeNames", new HashSet<>());
+
+    readAndUpdateMutesRcas.invoke(rcaController);
+    Assert.assertNull(Stats.getInstance().getMutedGraphNodes());
+
+    // Re-set back to initialComponentSet
+    Whitebox.setInternalState(ConnectedComponent.class, "nodeNames", initialComponentSet);
+  }
+
   @Test
   public void readAndUpdateMutesRcas() throws Exception {
```

Comment:
typo: `readAndUpdateMutedRcas` instead ?

Reply:
Fixed the typo.